extensions/cn-temp-mail/package.json

```
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "cn-temp-mail",
+  "title": "Temp Mail - CN",
+  "description": "It creates temporary email.",
```

```suggestion
  "name": "maildrop",
  "title": "Maildrop - Create Temporary Email",
  "description": "Provides free disposable e-mail addresses for use in web forms, app signups, or any other place you'd like to protect your privacy.",
```
And change folder name